script.js

```diff
@@ -0,0 +1,129 @@
+  },
+  {
+    id: 2,
+    personImg: './images/FeliciaAgoka.png',
+    personName: 'Dr. Felicia Agoka',
+    placeofWork: '37 Military Hospital',
+    duties: 'Cares for the whole family from children to the elderly, giving basic care. She does routine check-ups and screening tests, attending to ongoing medical conditions.',
+  },
+  {
+    id: 3,
+    personImg: './images/FrankAdjei.png',
+    personName: 'Dr. Frank Adjei',
+    placeofWork: 'Okomfo Anokye Teaching Hospital',
+    duties: 'An endocrinologist who treats the endocrine system. The glands producing and secreting hormones that regulate and control almost all of the bodys functions.',
+  },
+  {
+    id: 4,
+    personImg: './images/PriscillaAryee.png',
+    personName: 'Dr. Priscilla Aryee',
+    placeofWork: 'Korle-Bu Teaching Hospital',
+    duties: 'Infectious Disease Specialists who diagnose and treat infections in any part of your body, like fevers, Lyme disease, pneumonia, tuberculosis, and HIV and AIDS.',
+  },
+  {
+    id: 5,
+    personImg: './images/MariamOsei.png',
+    personName: 'Mrs Mariam Osei',
+    placeofWork: 'Fiber Systems Engineer at Vodafone Ghana',
+    duties: 'She designs, develops, and implements systems and databases to access and store geospatial data, analyze spatial and non-spatial information.',
+  },
+];
+
+speakers.innerHTML = `
+<div class="experts-details">
+<img src="${speakerData[0].personImg}" alt="Dr Annah" />
+<div class="experts-info">
+  <h3 class="experts-name">${speakerData[0].personName}</h3>
+  <p class="experts-place">${speakerData[0].placeofWork}</p>
+  <p class="experts-achieve">${speakerData[0].duties}
+  </p>
+</div>
+</div>
+
+<div class="experts-details">
+<img src="${speakerData[1].personImg}" alt="Dr Annah" />
+<div class="experts-info">
+  <h3 class="experts-name">${speakerData[1].personName}</h3>
+  <p class="experts-place">${speakerData[1].placeofWork}</p>
+  <p class="experts-achieve">${speakerData[1].duties}
+  </p>
+</div>
+</div>
+
+<div class="experts-details">
+<img src="${speakerData[2].personImg}" alt="Dr Annah" />
+<div class="experts-info">
+  <h3 class="experts-name">${speakerData[2].personName}</h3>
+  <p class="experts-place">${speakerData[2].placeofWork}</p>
+  <p class="experts-achieve">${speakerData[2].duties}
+  </p>
+</div>
+</div>
+
+<div class="experts-details">
+<img src="${speakerData[3].personImg}" alt="Dr Annah" />
+<div class="experts-info">
+  <h3 class="experts-name">${speakerData[3].personName}</h3>
+  <p class="experts-place">${speakerData[3].placeofWork}</p>
+  <p class="experts-achieve">${speakerData[3].duties}
+  </p>
+</div>
+</div>
+
+<div class="experts-details">
+<img src="${speakerData[4].personImg}" alt="Dr Annah" />
+<div class="experts-info">
+  <h3 class="experts-name">${speakerData[4].personName}</h3>
+  <p class="experts-place">${speakerData[4].placeofWork}</p>
+  <p class="experts-achieve">${speakerData[4].duties}
+  </p>
+</div>
+</div>
```

Comment:
- [ ] Please adjust the image size of your speaker on the mobile version to match the looks to the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)


| Cindy Shin design    | Your design |
| ------------- | ------------- |
| <img width="373" alt="Screenshot 2023-04-21 at 1 29 29 AM" src="https://user-images.githubusercontent.com/44798044/233480742-28ddd915-d343-420d-9a62-29ce9c843b20.png">| <img width="373" alt="Screenshot 2023-04-20 at 11 36 17 PM" src="https://user-images.githubusercontent.com/44798044/233480591-0e00878d-edb2-4417-9e4a-6672c98e3a00.png">|

- [ ] Kindly add the checker pattern in the background of each image just like in the [design guidelines.](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) Also note that the first paragraph (Place of work) is italic in the guidelines and also there is a small line after that so please adjust your website's styling according to the guidelines. Please also adjust the font-style and font-sizes of the speaker. cards details 

| Cindy Shin design    | Your design |
| ------------- | ------------- |
| <img width="385" alt="Screenshot 2023-04-20 at 11 41 27 PM" src="https://user-images.githubusercontent.com/44798044/233480101-6859d71c-ab95-4c5c-847d-d071b609238f.png">| <img width="724" alt="Screenshot 2023-04-20 at 11 38 25 PM" src="https://user-images.githubusercontent.com/44798044/233480158-1364e973-0871-4406-8666-eec6ee51c946.png">|

Reply:
Thanks for the reply, but I have a little issue with this review. This is not a Figma design so extracting the pattern is kind of difficult. I suggest you send it to me if you have it.

Reply:
Hi @Theodoraldo Well you can create one using figma or download any checker pattern it doesn't really have to be an exact match but there should be a checker pattern behind the images. Click the link below:

https://user-images.githubusercontent.com/44798044/233597017-dba44233-a8ca-4d89-9936-be3e8c956297.svg